### PR TITLE
Check-436: Run tests without configurator

### PR DESCRIPTION
**Pull request by danielevalverde**

### CODE REVIEW

File: app/models/concerns/team_import.rb [hunk 1]
```diff
@@ -4,33 +4,33 @@
 module TeamImport
```
Comment (caiosba) [inline app/models/concerns/team_import.rb:4]: Why you commented just some lines and not the whole file? Are the uncommented lines used for anything else that is not related to the `TeamImport` feature?

File: config/config.yml.example [hunk 2]
```diff
@@ -68,20 +68,20 @@ development: &default
   # Generate a key with SecureRandom to enable two-factor authentication (2FA) `SecureRandom.hex(20)`
   # If blank the QRCode won't be generated and the 2FA can't be enabled
   #
-  two_factor_key: # '<Google two factor key>'
+  two_factor_key: "a2ab0425f2cfa4243110e343bc865653aea941b91819e754c9a82a0115266add" # '<Google two factor key>'
```
Comment (caiosba) [inline config/config.yml.example:71]: This is just a random string not used anywhere else, right? :)

Comment (danielevalverde) [reply config/config.yml.example:71]: Yes, it was generated using `SecureRandom`
